requirements.txt

```diff
@@ -9,13 +9,11 @@ django-extensions==1.6.1
 django-pipeline==1.6.8
 django-wkhtmltopdf==3.0.0
 ecdsa==0.13
-enum34==1.0.4
 Flask==0.10.1
-functools32==3.2.3.post2
 futures==3.0.5
 gevent==1.2.1
 gnureadline==6.3.3
-grako==3.8.1
+grako==3.10.1
```

Comment:
Please remove pkg-resources=0.0.0 - apparently this is a bug in ubuntu
This breaks installing deps from requirements.txt, at least on non-ubuntu https://stackoverflow.com/questions/39577984/what-is-pkg-resources-0-0-0-in-output-of-pip-freeze-command